test/builder-api/src/origins.js

```diff
@@ -257,6 +257,44 @@ describe('Related Origin API functions', function () {
       });
   });
 
+    describe('Origin Departure', function () {
+        it('requires authentication', function(done) {
+            request.post('/depot/origins/umbrella/depart')
+            .expect(401)
+                .end(function (err, res) {
+                    expect(res.text).to.be.empty;
+                    done(err)
+                });
+        });
+        it('cannot be departed by its owner', function(done) {
+          request.post('/depot/origins/umbrella/depart')
+            .set('Authorization', global.boboBearer)
+            .expect(403)
+              .end(function (err, res) {
+                expect(res.text).to.be.empty;
+                done(err);
+              });
+        });
+        it('must already be member of origin', function(done) {
+          request.post('/depot/origins/umbrella/depart')
+            .set('Authorization', global.mystiqueBearer)
+            .expect(422)
+                .end(function (err, res) {
+                    expect(res.text).to.be.empty;
+                    done(err);
+                });
+        });
+        it('succeeds', function(done) {
+            request.post('/depot/origins/umbrella/depart')
+            .set('Authorization', global.weskerBearer)
+            .expect(204)
+                .end(function (err, res) {
+                    expect(res.text).to.be.empty;
+                    done(err)
+                });
+        });
+    });
+
```

Comment:
Add a check that user is no longer a member of the origin after the depart succeeds?